src/compiler/types.ts

```diff
@@ -2734,6 +2736,52 @@ namespace ts {
         resolveModuleNames?(moduleNames: string[], containingFile: string): ResolvedModule[];
     }
 
+    /* @internal */
+    export const enum TransformFlags {
+        // Facts
+        // - Flags used to indicate that a node or subtree contains syntax that requires transformation.
+        TypeScript = 1 << 0,
+        ContainsTypeScript = 1 << 1,

```

Comment:
What is a context in which `ContainsTypeScript` doesn't suffice for `TypeScript` itself.